scripts/node_control_tool.py

```diff
@@ -14,30 +19,41 @@ def compose_cmd(cmd):
 
 def call_upgrade_script(version):
     import subprocess
-    print('Upgrading sovrin node to version %s, test_mode %d ' % (version, int(test_mode)))
-    try:
-        if test_mode:
-            retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node_test', version]), shell=True)
-        else:
-            retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node', version]), shell=True)
-        if retcode != 0:
-            print('Upgrade failed')
-    except:
-        print('Something went wrong with calling upgrade script')
+    logger.info('Upgrading sovrin node to version %s, test_mode %d ' % (version, int(test_mode)))
+    if test_mode:
+        retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node_test', version]), shell=True)
+    else:
+        retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node', version]), shell=True)
+    if retcode != 0:
+        msg = 'Upgrade failed: upgrade script returned {}'.format(retcode)
```

Comment:
Right now a steward should receive an email prior to the upgrade if he has email notifier installed. Then he should receive an email with the result of the upgrade once the node started. If there's a failure he will contact dev department with the logs